### PR TITLE
[Feature][ci size] Web Pay -  Bottom Sheet

### PR DESCRIPTION
**Pull request by jorGonzalez291292**
## Motivación y Contexto
Se agrega flujo de alta de tarjeta de débito con webpay.

## Tipo de cambio (para el release manager)
- [ ] Breaking change (Fix o feature que cambia una funcionalidad existente o rompe firmas)

- [ ] Mi cambio afecta a los integradores internos

### CODE REVIEW

**Review by cgaggino — APPROVED** [review]

**Comment by guchito9** [comment]
418K	./px-services/build/outputs/aar/services-release.aar
508K	./px-services/build/outputs/aar/services-debugBeta.aar
508K	./px-services/build/outputs/aar/services-debug.aar
57K	./testlib/build/outputs/aar/testlib-debug.aar
57K	./testlib/build/outputs/aar/testlib-debugBeta.aar
52K	./testlib/build/outputs/aar/testlib-release.aar
138K	./px-testlib/build/outputs/aar/testlib-debug.aar
138K	./px-testlib/build/outputs/aar/testlib-debugBeta.aar
123K	./px-testlib/build/outputs/aar/testlib-release.aar
1.8M	./px-checkout/build/outputs/aar/checkout-release.aar
2.0M	./px-checkout/build/outputs/aar/checkout-debug.aar
2.0M	./px-checkout/build/outputs/aar/checkout-debugBeta.aar
61K	./px-addons/build/outputs/aar/addons-debug.aar
52K	./px-addons/build/outputs/aar/addons-release.aar
61K	./px-addons/build/outputs/aar/addons-debugBeta.aar

**Comment by guchito9** [comment]
418K	./px-services/build/outputs/aar/services-release.aar
508K	./px-services/build/outputs/aar/services-debugBeta.aar
508K	./px-services/build/outputs/aar/services-debug.aar
57K	./testlib/build/outputs/aar/testlib-debug.aar
57K	./testlib/build/outputs/aar/testlib-debugBeta.aar
51K	./testlib/build/outputs/aar/testlib-release.aar
138K	./px-testlib/build/outputs/aar/testlib-debug.aar
138K	./px-testlib/build/outputs/aar/testlib-debugBeta.aar
123K	./px-testlib/build/outputs/aar/testlib-release.aar
1.8M	./px-checkout/build/outputs/aar/checkout-release.aar
2.0M	./px-checkout/build/outputs/aar/checkout-debug.aar
2.0M	./px-checkout/build/outputs/aar/checkout-debugBeta.aar
61K	./px-addons/build/outputs/aar/addons-debug.aar
52K	./px-addons/build/outputs/aar/addons-release.aar
61K	./px-addons/build/outputs/aar/addons-debugBeta.aar

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/ExpressPaymentFragment.java [hunk 1]
```diff
@@ -53,10 +61,11 @@
 import com.mercadopago.android.px.internal.features.pay_button.PayButton;
 import com.mercadopago.android.px.internal.features.pay_button.PayButtonFragment;
 import com.mercadopago.android.px.internal.features.security_code.SecurityCodeFragment;
-import com.mercadopago.android.px.internal.util.CardFormWithFragmentWrapper;
+import com.mercadopago.android.px.internal.util.CardFormWrapper;
 import com.mercadopago.android.px.internal.util.ErrorUtil;
 import com.mercadopago.android.px.internal.util.Logger;
 import com.mercadopago.android.px.internal.util.VibrationUtils;
+import com.mercadopago.android.px.internal.util.ViewUtils;
```
Comment (cgaggino) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/ExpressPaymentFragment.java:68]: quedó de más

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/add_new_card/OtherPaymentMethodFragment.java [hunk 2]
```diff
@@ -145,15 +174,30 @@ protected void loadImage(@NonNull final View view, @DrawableRes final int imageR
...
+        switch (initType) {
+            case STANDARD: {
+                final FragmentManager manager;
+                if (getParentFragment() != null && (manager = getParentFragment().getFragmentManager()) != null) {
+                    cardFormWrapper.getCardFormWithFragment()
+                        .start(manager, REQ_CODE_CARD_FORM, R.id.one_tap_fragment);
+                }
+                break;
+            }
+            case WEBPAY_TBK: {
```
Comment (cgaggino) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/add_new_card/OtherPaymentMethodFragment.java:187]: Podemos cambiarle el nombre a solo webpay? no dice mucho TBK

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/add_new_card/OtherPaymentMethodPresenter.java [hunk 3]
```diff
@@ -2,23 +2,19 @@
...
-        this.settingRepository = settingRepository;
-        this.trackingRepository = trackingRepository;
+    /* default */ OtherPaymentMethodPresenter(@NonNull final CardFormWrapper cardFormWrapper) {
+        this.cardFormWrapper = cardFormWrapper;
     }
 
     @Override
-    public void onAddNewCardSelected() {
-        getView().startCardForm(new CardFormWithFragmentWrapper(settingRepository, trackingRepository));
+    public void onAddNewCardSelected(final CardFormInitType initType) {
```
Comment (cgaggino) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/add_new_card/OtherPaymentMethodPresenter.java:17]: Es nonnull?

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/add_new_card/OtherPaymentMethodFragment.java (hunk 2 above)
Comment (jorGonzalez291292) [reply px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/add_new_card/OtherPaymentMethodFragment.java:187]: Si se puede, lo cambio a `WEB_PAY`

**Review by cgaggino — APPROVED** [review]

**Comment by guchito9** [comment]
418K	./px-services/build/outputs/aar/services-release.aar
508K	./px-services/build/outputs/aar/services-debugBeta.aar
508K	./px-services/build/outputs/aar/services-debug.aar
57K	./testlib/build/outputs/aar/testlib-debug.aar
57K	./testlib/build/outputs/aar/testlib-debugBeta.aar
51K	./testlib/build/outputs/aar/testlib-release.aar
138K	./px-testlib/build/outputs/aar/testlib-debug.aar
138K	./px-testlib/build/outputs/aar/testlib-debugBeta.aar
123K	./px-testlib/build/outputs/aar/testlib-release.aar
1.8M	./px-checkout/build/outputs/aar/checkout-release.aar
2.0M	./px-checkout/build/outputs/aar/checkout-debug.aar
2.0M	./px-checkout/build/outputs/aar/checkout-debugBeta.aar
61K	./px-addons/build/outputs/aar/addons-debug.aar
52K	./px-addons/build/outputs/aar/addons-release.aar
61K	./px-addons/build/outputs/aar/addons-debugBeta.aar

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/add_new_card/OtherPaymentMethodPresenter.java (hunk 3 above)
Comment (jorGonzalez291292) [reply px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/add_new_card/OtherPaymentMethodPresenter.java]: Si, agrego el @NonNull.

**Comment by guchito9** [comment]
418K	./px-services/build/outputs/aar/services-release.aar
508K	./px-services/build/outputs/aar/services-debugBeta.aar
508K	./px-services/build/outputs/aar/services-debug.aar
57K	./testlib/build/outputs/aar/testlib-debug.aar
57K	./testlib/build/outputs/aar/testlib-debugBeta.aar
51K	./testlib/build/outputs/aar/testlib-release.aar
138K	./px-testlib/build/outputs/aar/testlib-debug.aar
138K	./px-testlib/build/outputs/aar/testlib-debugBeta.aar
123K	./px-testlib/build/outputs/aar/testlib-release.aar
1.8M	./px-checkout/build/outputs/aar/checkout-release.aar
2.0M	./px-checkout/build/outputs/aar/checkout-debug.aar
2.0M	./px-checkout/build/outputs/aar/checkout-debugBeta.aar
61K	./px-addons/build/outputs/aar/addons-debug.aar
52K	./px-addons/build/outputs/aar/addons-release.aar
61K	./px-addons/build/outputs/aar/addons-debugBeta.aar

**Comment by guchito9** [comment]
418K	./px-services/build/outputs/aar/services-release.aar
508K	./px-services/build/outputs/aar/services-debugBeta.aar
508K	./px-services/build/outputs/aar/services-debug.aar
57K	./testlib/build/outputs/aar/testlib-debug.aar
57K	./testlib/build/outputs/aar/testlib-debugBeta.aar
51K	./testlib/build/outputs/aar/testlib-release.aar
138K	./px-testlib/build/outputs/aar/testlib-debug.aar
138K	./px-testlib/build/outputs/aar/testlib-debugBeta.aar
123K	./px-testlib/build/outputs/aar/testlib-release.aar
1.8M	./px-checkout/build/outputs/aar/checkout-release.aar
2.0M	./px-checkout/build/outputs/aar/checkout-debug.aar
2.0M	./px-checkout/build/outputs/aar/checkout-debugBeta.aar
61K	./px-addons/build/outputs/aar/addons-debug.aar
52K	./px-addons/build/outputs/aar/addons-release.aar
61K	./px-addons/build/outputs/aar/addons-debugBeta.aar

**Pull request merged**